src/Microsoft.ML.Data/DataLoadSave/Text/TextLoader.cs

```diff
@@ -1064,15 +1065,15 @@ private bool HasHeader
         /// </summary>
         /// <param name="env">The environment to use.</param>
         /// <param name="columns">Defines a mapping between input columns in the file and IDataView columns.</param>
-        /// <param name="separatorChar"> The character used as separator between data points in a row. By default the tab character is used as separator.</param>
+        /// <param name="separators"> The character used as separator between data points in a row. {'\t'} will be used if not specified.</param>
```

Comment:
>{'\t'} [](start = 100, length = 6)

what is wrong with `tab` ?

Reply:
`tab` is scalar, not an array.

---
In reply to: [259558700](https://github.com/dotnet/machinelearning/pull/2702#discussion_r259558700) [](ancestors = 259558700)

Reply:
ok, what is difference from user standpoint between array with one element and scalar?

---
In reply to: [259559041](https://github.com/dotnet/machinelearning/pull/2702#discussion_r259559041) [](ancestors = 259559041,259558700)

Reply:
Their types are obviously different, right?

---
In reply to: [259559198](https://github.com/dotnet/machinelearning/pull/2702#discussion_r259559198) [](ancestors = 259559198,259559041,259558700)